Description: Fix for an issue in the MultiMail man page.
 The "'#'" at the beginning of line 363 causes a Lintian warning about a
 macro not being defined but it is actually referring to the "#" character
 as being used to indicate a comment line in the description of the file
 used for the MultiMail configuration and it is a coincidence that it is at
 the beginning of the line. Patch to include coding for a zero width space.
Author: Robert James Clay <jame@rocasa.us>
Last-Update: <2013-05-04>

--- multimail-0.49.orig/mm.1
+++ multimail-0.49/mm.1
@@ -1,4 +1,4 @@
-.TH MultiMail 1 "July 19, 2007"
+.TH MultiMail 1 "May 4, 2013"
 .SH NAME
 mm - offline mail reader for Blue Wave, QWK, OMEN, SOUP and OPX packets
 .SH SYNOPSIS
@@ -360,7 +360,7 @@ The default directory for saving message
 The config file (see above) is a plain text file with a series of values,
 one per line, in the form "KeyWord: Value". The case of the keywords is
 not signifigant. Additional, comment lines may be present, starting with
-'#'; you can remove these or add your own. (But note that the comments are
+\&'#'; you can remove these or add your own. (But note that the comments are
 replaced by the defaults when you upgrade to a new version.) If any of the
 keywords are missing, default values will be used.
 
